docs/components/Menu/Web.stories.tsx

```diff
@@ -51,7 +51,9 @@ Horizontal.args = {
 };
 
 const CustomActivatorTemplate: ComponentStory<typeof Menu> = args => (
-  <Menu {...args} activator={<Button label="My Fancy Menu" />} />
+  <div style={{ width: "200px", height: "100px", overflow: "hidden" }}>
```

Comment:
more code that can be removed. only exists to illustrate the overflow case ain't broken no more.